src/helpers/ens.ts

```diff
@@ -625,6 +625,11 @@ const getENSExecutionDetails = async ({
 
 const getENSRecordKeys = () => Object.keys(ENS_RECORDS);
 const getENSRecordValues = () => Object.values(ENS_RECORDS);
+export const getENSRecordValuesForProfile = () => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { avatar, cover, description, ...rest } = ENS_RECORDS;
```

Comment:
does showing addresses too make sense?

Reply:
might need to truncate it

Reply:
at some point i'd say so, is blocked for now tho https://github.com/rainbow-me/rainbow/pull/3263/files#diff-3a21e0edcddda5a805eb35d73d6f7367892e4b51b2c5205dab3b06912daa9a54R97 we're only passing `records`, we'd need to pass `coinAddresses`

the only reason i didn't add it was that we don't have icons for the addresses i'l ask christian

Reply:
Yeah that makes sense. Maybe we don't need to do it then.

Also, does it make sense for this to be in this file? Since it's only being used in one place, what if we keep it in `ProfileSheetHeader` to avoid context switching?

Reply:
i moved it out, waiting for the "order records" ticket we have that's not defined yet i think it makes more sense to do whatever we want to do there 👌